### PR TITLE
HEE-163: Change the term Listing page to be Landing page

### PR DESCRIPTION
**Pull request by thojava**

### CODE REVIEW

File: repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-/content/documents/lks/listing-pages:
```
Comment (smanivasag) [inline repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml:1]: Could we add an Content Action [`hcm-actions.yaml`] in order to get rid of existing `listing-pages` folder and it's descendants from local/brCloud dev environments ?

**Review by smanivasag — COMMENTED** [review]
Hi @thojava - Thanks for the PR!

Does this change mean, we need to update the corresponding channel component `Listing Page` to `Landing Page` ?

File: repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml (hunk 1 above)
Comment (thojava) [reply repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml:1]: @smanivasag I have updated that.

Comment (smanivasag) [reply repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml:1]: Thanks @thojava

**Review by smanivasag — APPROVED** [review]

**Pull request merged**